### ISSUE TITLE
Can't coy gist from Windows

### ISSUE THREAD

**Issue by FiretronP75** [keybase/client#11814]
Windows app gave me a gist to copy to github, but it is impossible to copy, which means it is then impossible to paste.

**Comment by wis** [comment]
same situation happened 2 days ago, I just selected it and dragged it to a text editor/notepad then copied it.

**Comment by DatGinger** [comment]
I have also had the same problem, both on the windows and Linux client.  I was also unable to copy to text editors in either os. The app gives the copy drop down, but there is no output to either system clipboard.

**Comment by DatGinger** [comment]
Yes that indeed worked.  For some reason in my head text is text and nary an object shall meet.  I am however, greener than moss so I am not sure if this is a bug or a feature.  Thank you

**Comment by janoulle** [comment]
I'm on Windows 10 Pro and tried dragging the text to Visual Studio Code & Notepad (didn't work). Any recommendations for other text editors I can try that'll support this dragging action?

**Comment by wis** [comment]
@janoulle Notepad++

**Comment by wis** [comment]
@FiretronP75 can you edit the typo in the title please? because there were few identical issues opened. 
because to Lucene/Elastic full text search "coy" != "copy"... and/or just too much laziness to read issue titles and scroll down a bit.
